src/components/MultiHopTrade/hooks/useSupportedAssets.tsx

```diff
@@ -10,15 +11,16 @@ export const useSupportedAssets = () => {
   const sellAsset = useAppSelector(selectSellAsset)
   const assetIds = useAppSelector(selectAssetIds)
   const sortedAssets = useAppSelector(selectAssetsSortedByMarketCapFiatBalanceAndName)
-  const { LifiSwap, ThorSwap } = useAppSelector(selectFeatureFlags)
+  const { LifiSwap, ThorSwap, ZrxSwap } = useAppSelector(selectFeatureFlags)
 
   const enabledSwappers = useMemo(() => {
     const result = []
     if (LifiSwap) result.push(lifi)
     if (ThorSwap) result.push(thorchain)
+    if (ZrxSwap) result.push(zrx)
     // TODO(woodenfurniture): add more swappers here
     return result
-  }, [LifiSwap, ThorSwap])
+  }, [LifiSwap, ThorSwap, ZrxSwap])
```

Comment:
Non-blocking but I remember these being renamed to the `swapper` vernacular somewhere else e.g `zrxSwapper`, if we want to do it here as well